metron-deployment/other-examples/README.md

```diff
@@ -0,0 +1,3 @@
+# Other Example Deployments
```

Comment:
What I would want to see in this readme is some kind of disclaimer, but I'm not sure what it would be.

"these examples have been vetted and tested by the metron community as of the time of entry.  Things may change however, and if there are problems with these steps, please log a jira"

Also, should there be a date on each example, to show when it was entered?
Should there be a revision log?